db/migrate/20221125173217_remove_enable_whispers_site_setting.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+class RemoveEnableWhispersSiteSetting < ActiveRecord::Migration[7.0]
+  def up
+    # If enable_whispers was true, insert whispers_allowed_groups or add
+    # staff group to whispers_allowed_groups.
+    execute <<~SQL
+      INSERT INTO site_settings(name, data_type, value, created_at, updated_at)
+      SELECT 'whispers_allowed_groups', '20', '3', created_at, NOW()
```

Comment:
We should probably explain that we're adding the group `3`, which is the staff group, because the previous behaviour had a bypass for all staff members.